### ISSUE TITLE
Screen readers are not narrating the descriptive information for 'here' link while navigating using tab key.

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5100]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Screen reader users will be impacted as focus moves to hidden element and announces Unnecessary information which might confuse users. The user will get confused, and the issue will lead to disorientation. 

### What browsers and screen readers do this issue affect?

Windows: Chrome with NVDA

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://microsoft.sharepoint.com/teams/MCAPSAcademy/SitePages/Request-a-Contributor---POC.aspx?xsdata=MDV8MDF8ZGJoYWt0YUBtaWNyb3NvZnQuY29tfGUzYWVmOTc1NDViMTQzYTg4Y2ZhMDhkYmM1YmQ5YjI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2MzgzMjExOTAzODMzODc3NDF8VW5rbm93bnxUV0ZwYkdac2IzZDhleUpXSWpvaU1DNHdMakF3TURBaUxDSlFJam9pVjJsdU16SWlMQ0pCVGlJNklrMWhhV3dpTENKWFZDSTZNbjA9fDMwMDB8fHw%3D&sdata=SSs2MzJWZWN5M2JyTVZGeWp4c0VjbkJlNHIvZEpueXNreC9IZktzSkMxTT0%3D&OR=Teams-HL&CT=1696572496023&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiIyNy8yMzA5MTQxNzUwNSIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3D%3D

### How to reproduce the issue?

Step 1: Open the above URL in Edge Chromium Browser. > Sign in with valid credentials
Step 2: Type 'How can I support someone on my team who is contributing through the program? in edit field.
Step 3: Verify whether Screen readers are narrating the descriptive information for 'here' link while navigating using tab key or not.

### What do you expect?

Screen readers should narrate the descriptive information for 'here' link while navigating using tab key.

### What actually happened?

Screen readers are not narrating the descriptive information for 'here' link while navigating using tab key.
Screen reader only announces it as: link here

### Do you have any screenshots or recordings to repro the issue?

<img width="960" alt="MAS 1 3 1-Screen readers are not narrating the descriptive information for 'here' link while navigating using tab key" src="https://github.com/microsoft/BotFramework-WebChat/assets/117266902/15a64168-af85-4298-8c55-99d0f809a739">

### MAS reference

https://aka.ms/MAS2.4.4

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev3;#MAS1.3.1;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;

**Comment by Kirank915** [comment]
Bug copied from MCAPS: [ISSUE 1767719](https://dev.azure.com/worldwidelearning/WWL%20Accessibility/_workitems/edit/1767719)

**Comment by compulim** [comment]
I cannot open the SharePoint link.

Could you open F12 and copy and paste the HTML code around the "here" link?

**Comment by Kirank915** [comment]
@compulim we are checking with MCAPS team for further details. We will update here as soon as we hear back from them
